reo/nested_inputs.py

```diff
@@ -404,10 +561,56 @@ def list_of_int(input):
           "replacement_sets": electric_tariff_possible_sets,
           "description": "Label attribute of utility rate structure from <a href='https: //openei.org/services/doc/rest/util_rates/?version=3' target='blank'>Utility Rate Database API</a>"
         },
-         "emissions_factor_series_lb_CO2_per_kwh": {
-          "type": "list_of_float",
+        "emissions_factor_series_lb_CO2_per_kwh": {
+          "type": ["list_of_float", "float"],
           "description": "Carbon Dioxide emissions factor over all hours in one year. Must be hourly (8,760 samples), 30 minute (17,520 samples), or 15 minute (35,040 samples).",
```

Comment:
@t-kwasnik this description for `emissions_factor_series_lb_CO2_per_kwh` does not line up with the allowed type of `float`; are we also allowing a single, constant value as an input?

Reply:
@NLaws yes, similar to wholesale_rate_us_dollars_per_kwh it can be a single value that gets turned into a annual timeseries, or an annual timeseries

Reply:
Ok can you please update the description?